prow/cmd/gerrit/main.go

```diff
@@ -36,13 +36,20 @@ import (
 
 type options struct {
 	cookiefilePath   string
-	configPath       string
-	jobConfigPath    string
+	config           flagutil.ConfigOptions
 	projects         client.ProjectsFlag
 	lastSyncFallback string
 }
 
 func (o *options) Validate() error {
+	if err := o.config.Validate(false); err != nil {
+		return err
+	}
+
+	if o.config.ConfigPath == "" {
```

Comment:
Isn't this checked by the `ConfigOptions`?

Reply:
> Isn't this checked by the `ConfigOptions`?

Not at the moment.  Discussion about that starts [here][1].

[1]: https://github.com/kubernetes/test-infra/pull/9701#discussion_r236778941